bt buổi 10/main.js

```diff
@@ -0,0 +1,96 @@
+function getMultiplication() {
+    for(let i = 0; i < 10; i++) {
+        console.log(6 * i)
+    }
+}
+
+// Bài 1
+const printMultiplication = number => {
+    for(let i = 0; i < 10; i++) {
+        console.log(6 * i)
+    }
+}
+
+// Bài 2
+const printEvenNumbers = number => {
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+            if (i % 2 === 0) console.log(i)
+    } 
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
+}
+
+// Bài 3
+const getSum = number => {
+    let sum = 0
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+             sum += i
+    } console.log(sum)
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
+}
+
+// Bài 4
+const getFactorial = number => {
+    let factorial = 1
+    if (number >= 1 && number <= 30) {
+        for(let i = 1; i <= number; i++) {
+            factorial *= i
+    } console.log(factorial)
+    } else console.log('n chỉ nhận giá trị từ 1 đến 30')
```

Comment:
```suggestion
    if (number >= 1 && number <= 30) {
      for(let i = 1; i <= number; i++) {
          factorial *= i
      }
      console.log(factorial)
    } else {
      console.log('n chỉ nhận giá trị từ 1 đến 30')
    }
```

Ngoài ra khi viết, thì hạn chế `if else` đặc biệt là `if else` hoặc các block lồng nhau
Hãy viết như thế này sẽ dễ nhìn hơn:
```javascript
  if (number < 1 || number > 30) return;

  let factorial = 1
  for (let i = 1; i <= number; i++) {
    factorial *= i
  }
  return factorial
```

Và đề bài cũng cần return chứ ko phải `log`